src/elements/flip-card/flip-card-summary/flip-card-summary.scss

```diff
@@ -42,7 +42,7 @@
   padding-block: var(--sbb-spacing-responsive-s) var(--sbb-spacing-responsive-xs);
   margin: 0;
   grid-area: 1 / 1 / 2 / 2;
-  transform: translateY(var(--sbb-flip-card-title-translate-y-hover, #{sbb.px-to-rem-build(0)}));
+  transform: translateY(var(--sbb-flip-card-translate-title-y-hover, #{sbb.px-to-rem-build(0)}));
```

Comment:
I think this needs to be applied to all the slotted content of the summary and not only the title, although we normally only expect titles slotted. But consumers could also slot `<h5>` or whatever that should receive the hover effect too.